pkg/commands/global.go

```diff
@@ -22,24 +22,14 @@ func GlobalFlags() []cli.Flag {
 			Name:  "log-caller",
 			Usage: "log the caller (aka line number and file)",
 		},
-		&cli.BoolFlag{
-			Name:  "log-disable-color",
-			Usage: "disable log coloring",
-		},
-		&cli.BoolFlag{
-			Name:  "log-full-timestamp",
-			Usage: "force log output to always show full timestamp",
-		},
```

Comment:
Just curious, why were these options removed?

Reply:
They were only used by the previous log format that was unstructured. As far as I can tell, there are not analogous options for the JSON formatter.